### DISCUSSION TITLE
How do I set up sequential updates?

### DISCUSSION THREAD

**Discussion by zygopleural** [renovatebot/renovate#24410] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I want to be able to configure Renovate to sequentially update a specific package. e.g. if I am on version `1.2.3` and version `1.2.4` and `1.3.0` are available, I want it to only open the PR for version `1.2.4` and not open the PR for `1.3.0` until `1.2.4` is merged.

### What you tried so far.

Searching https://docs.renovatebot.com/configuration-options/ to no avail.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
If the 1.2.4 is missing then you can configure seprateMinorPatch=true to get it. It's not possible to _suppress_ the minor update in such a case automatically though.
